core/src/main/resources/hudson/model/UsageStatistics/footer-resources.js

```diff
@@ -0,0 +1,3 @@
+Behaviour.addLoadEvent(function() {
+    loadScript("https://usage.jenkins.io/usage-stats.js?${statData}");
```

Comment:
This won't work. Have you tested it at all?

Reply:
It needs to be tested interactively with a web browser in addition to the test automation.  Human evaluation of presentation of a web page sees things that the automated test assertions may not be checking.

Reply:
The `statData` is a java variable that is injected in the Jelly. If you are moving to a separate file, this value will be not provided to the JS file.

You can find more information in https://www.jenkins.io/doc/developer/security/xss-prevention/#passing-values-to-javascript.

But yeah, please test it manually ;)